c/zinfo.h

```diff
@@ -40,8 +40,8 @@
   from the original.
 */
 
-#ifndef INDEXER_H
-#define INDEXER_H
+#ifndef ZINFO_H
```

Comment:
Maybe instead `zinfo.h`, rename it `gzip_zinfo.h`? This way, we will not need to rename it when there will be the implementation to build Zinfo for other compression algorithms.

Reply:
The scope of this change was strictly about clarifying the naming around zinfo and checkpoints. I resisted the urge to add or remove the GZIP-ness from names of things, even in cases like this where--as you point out--it would improve the code to do so.

I think that that is going to be part of the work of adding the missing interfaces around compression to make the world safe for adding ZStandard support. That is going to be a much larger effort involving the creation of new `interface`s and moving source files--including these--to new directories.

Reply:
Sounds good.